package.json

```diff
@@ -122,5 +122,13 @@
   "engines": {
     "node": ">=6.3.0"
   },
-  "browserslist": "last 3 versions, >1%"
+  "browserslist": "last 3 versions, >1%",
+  "contributors": [
```

Comment:
It is probably better if we do it like cytoscape.js:

https://github.com/cytoscape/cytoscape.js/blob/master/package.json#L19

where names, emails, links have their own fields for each contributor

Reply:
I updated the file so emails, names, and links have their own fields.